File: JavaScript/algorithm/src/algorithm/연속부분수열합의개수/test2.js
const arr = [1, 2, 3];
const arr2 = [];

let start = 0;
let end = 0;
while (true) {
  let num = arr[start];
  while (start !== end) {
    num += arr[end];
  }
}
console.log(arr2);

// [1], [2], [3], [1, 2], [2, 3], [3, 1], [1, 2, 3]

File: python/algorithm/everything_algorithm/algorithm_study/level1/공원산책/test2.py
# ["SOO","OXX","OOO"]

arr = ["SOO","OXX","OOO"]
arr2 = []
for i in range(len(arr)):
    print('arr[i] : ', arr[i])
    j = 0

    for s in arr[i]:
        if 'X' in s:
            arr2.append([i, j])

        j += 1

print(arr2)
# print(x, y)